### ISSUE TITLE
Dumbbell plot

### ISSUE THREAD

**Issue by borntofrappe** [borntofrappe/learning-d3#43]
Map data with a [dumbbell plot](https://datavizproject.com/data-type/dumbbell-plot/)

**Comment by borntofrappe** [comment]
In terms of data consider [life expectancy in the EU](https://ec.europa.eu/eurostat/cache/digpub/demography/bloc-2c.html?lang=en), either mapping the gap between two years or the male-female split.

**Comment by borntofrappe** [comment]
Try to also show the average and standard variation as [in this more complex demo](https://r-graph-gallery.com/web-extended-dumbbell-plot-ggplot2.html).

**Comment by borntofrappe** [comment]
I tried to highlight the data points with symbols from `d3.symbols`.

![Dumbell Symbols](https://user-images.githubusercontent.com/33316703/208773796-cec1d21d-c63d-4662-8d29-0987d1afd6bd.jpg)

Ultimately, however, I decided to roll back to circles and recreate the visual mentioned in the previous message [from r-graph-gallery.com](https://r-graph-gallery.com/web-extended-dumbbell-plot-ggplot2.html). I'm quite satisfied with the end result.

![Dumbbell Plot](https://user-images.githubusercontent.com/33316703/208773963-7ae0fc96-812e-46e8-8b5f-1545aea97a85.jpg)
